src/site/twitch.tv/modules/emote-menu/EmoteMenuTab.vue

```diff
@@ -29,6 +29,13 @@
 					>
 						<div class="set-sidebar-icon">
 							<img v-if="es.owner && es.owner.avatar_url" :src="es.owner.avatar_url" />
+							<div v-else-if="es.provider === 'EMOJI'" class="emoji-group">
+								<SingleEmoji
+									:id="emojiGroupIcons[emojiCategories.indexOf(es.name)]"
+									:alt="es.name"
+									:style="{ width: `100%`, height: `100%` }"
+								/>
+							</div>
```

Comment:
nit: it makes more sense to invert this condition; while it's unlikely `owner` would ever be defined on an emoji set, if it ever was then this means the special emoji case wouldn't render.

```suggestion
							<div v-if="es.provider === 'EMOJI'" class="emoji-group">
								<SingleEmoji
									:id="emojiGroupIcons[emojiCategories.indexOf(es.name)]"
									:alt="es.name"
									:style="{ width: `100%`, height: `100%` }"
								/>
							</div>
							<img v-else-if="es.owner && es.owner.avatar_url" :src="es.owner.avatar_url" />
```

Reply:
Changed the order in 1dda7d0 .
However, I don't know if this is the right thing to do, because you have to specify a link in the owner that you probably want to display. It should be fine either way for this special scenario and I trust you have more experience for this stuff.